apps/game_backend/lib/game_backend/arena_servers/arena_server.ex

```diff
@@ -0,0 +1,24 @@
+defmodule GameBackend.ArenaServers.ArenaServer do
+  @moduledoc """
+  ArenaServers
+  """
+  use GameBackend.Schema
+  import Ecto.Changeset
+
+  schema "arena_servers" do
```

Comment:
You can add it here (see comment below):
```suggestion
  @derive Jason.Encoder
  schema "arena_servers" do
```